examples/image_ocr.py

```diff
@@ -0,0 +1,433 @@
+# This example uses a convolutional stack followed by a recurrent stack
+# and a CTC logloss function to perform optical character recognition
+# of generated text images. I have no evidence of whether it actually 
+# learns general shapes of text, or just is able to recognize all
+# the different fonts thrown at it...the purpose is more to demonstrate CTC
+# inside of Keras.  Note that the font list may need to be updated
+# for the particular OS in use. 
+# 
+# This starts off with easy 5 letter words. After 10 or so epochs, CTC  
+# learn translataional invariance, so longer words and groups of words 
+# with spaces are gradually fed in. 
+# 
+# Due to the use of a dummy loss function, Theano requires the following flags:
+# "on_unused_input='ignore'
+#
+# 
+# Created by Mike Henry
+# https://github.com/mbhenry/
+
+import sys
+import random
+import os
+import itertools
+import re
+import datetime
+import cairocffi as cairo
+import editdistance
```

Comment:
This example requires 2 extra packages. These should be listed in the docstring, so that the user can install them before they run the script.